testsuite/features/step_definitions/command_steps.rb

```diff
@@ -5,6 +5,27 @@
 require 'timeout'
 require 'nokogiri'
 
+# Sanity checks
+
+Then(/^"([^"]*)" should have a FQDN$/) do |host|
+  node = get_target(host)
+  result, return_code = node.run('hostname -f')
+  raise 'cannot determine hostname' unless return_code.zero?
+  raise 'hostname is not fully qualified' unless result.delete!("\n") == node.full_hostname
+end
+
+Then(/^"([^"]*)" should communicate with the server$/) do |host|
+  node = get_target(host)
+  node.run("ping -c1 #{$server.full_hostname}")
+  $server.run("ping -c1 #{node.full_hostname}")
+end
+
+Then(/^it should be possible to download file "([^"]*)"$/) do |file|
+  $server.run("cd /tmp && wget --no-check-certificate #{file}")
+end
```

Comment:
If we don't need the file, just throw it to /dev/null from a curl.
Also be caution, when we update clients, sle15sp1 has not wget installed by default

Reply:
it's small files... will not change much. But OK.

Are we sure curl is on all OS versions?

Reply:
nope 🤷🏼‍♂️😂

Reply:
from past experience, I had less problems with wget than with curl... maybe wisdom is to keep unchanged and not change.